### ISSUE TITLE
causing the problem of explode of loss value ..﻿

### ISSUE THREAD

**Issue by navinkeshava** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#96]
WARNING:root:Variable [FirstStageFeatureExtractor/InceptionV2/Conv2d_1a_7x7/BatchNorm/beta/Momentum] is not available in checkpoint

is above given error is going to cause any problem with training , and is causing the problem of explode of loss value ..﻿
